src/org/jitsi/impl/neomedia/codec/video/h264/DePacketizer.java

```diff
@@ -519,6 +537,105 @@ else if (nal_unit_type == 28) // FU-A Fragmentation unit (FU)
     }
 
     /**
+     * Returns true if the buffer contains a H264 key frame at offset
+     * <tt>offset</tt>.
+     *
+     * @param buff the byte buffer to check
+     * @param off the offset in the byte buffer where the actuall data starts
+     * @param len the length of the data in the byte buffer
+     * @return true if the buffer contains a H264 key frame at offset
+     * <tt>offset</tt>.
+     */
+    public boolean isKeyFrame(byte[] buff, int off, int len)
+    {
+      if (buff == null || len < 1 || buff.length < len)
```

Comment:
Actually @bgrozev noticed that the condition I gave you is wrong, it should look like this

    if (buf == null ||  buf.length < off + Math.max(len, 1 /* required array length for this method to work correctly */))
    {
        return false;
    }